webapp/components/compat-2021.js

```diff
@@ -400,54 +553,63 @@ class Compat2021FeatureChart extends PolymerElement {
           height: 350px;
           margin: 0 auto;
         }
-      </style>
 
-      <!-- TODO: replace with google-chart polymer element? -->
+        paper-dialog {
+          max-width: 600px;
+        }
+      </style>
       <div id="failuresChart" class="chart"></div>
+
+      <paper-dialog with-backdrop id="firefoxNightlyDialog">
+        <h2>Firefox Nightly Changelogs</h2>
+        <div>
+          Nightly builds of Firefox are all given the same sub-version,
+          <code>0a1</code>, so we cannot automatically determine the changelog.
+          To find the changelog of a specific Nightly release, locate the
+          corresponding SHAs on the
+          <a href="https://hg.mozilla.org/mozilla-central/firefoxreleases"
+             target="_blank">release page</a>, enter them below, and click "Go".
+          <paper-input id="firefoxNightlyDialogFrom" label="From SHA"></paper-input>
+          <paper-input id="firefoxNightlyDialogTo" label="To SHA"></paper-input>
```

Comment:
```suggestion
          corresponding revisions on the
          <a href="https://hg.mozilla.org/mozilla-central/firefoxreleases"
             target="_blank">release page</a>, enter them below, and click "Go".
          <paper-input id="firefoxNightlyDialogFrom" label="From revision"></paper-input>
          <paper-input id="firefoxNightlyDialogTo" label="To revision"></paper-input>
```

Matching the terminology of the linked page.